packages/common/utils/src/Numbers.spec.ts

```diff
@@ -37,6 +38,11 @@ describe('Numbers', () => {
     expect(commaizeNumber('100000000')).toEqual('100,000,000');
   });
 
+  test('decommaizeNumber', () => {
+    expect(decommaizeNumber('13,209,802')).toEqual(13209802);
+    expect(decommaizeNumber('1234,1,,234')).toEqual(12341234);
```

Comment:
Fix this please.

```typescript
expect(decommaizeNumber('12,341,234')).toEqual(12341234);
```

Reply:
I wanted to show that it works well even if there are multiple comma in a string, so is there no need to verify it?🙋🏼‍♂️

Reply:
I think this is not our intended behavior (we expect only string literals which are correctly commaized), so I think excluding this from our test case is better. (Test cases should only include what we intend the function to do.)

Reply:
Thank you for your answer!
Only the test code was wrote to verify the intention of the function author.
It was an opportunity to think about it again Thank you so much!